azure-functions/acbs-function/package.json

```diff
@@ -38,14 +38,14 @@
   "devDependencies": {
     "babel-jest": "29.5.0",
     "babel-loader": "9.1.2",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.1",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-import": "^2.30.0",
     "jest": "^29.7.0"
   },
   "engines": {
-    "node": ">=20",
-    "npm": ">=10"
+    "node": ">=22.8.0",
```

Comment:
Quick note of Mike's comment in #devs slack is quite valid, this puts us ahead of LTS. I think being on LTS is normally the safer option in general but if we don't mind taking the small amount of risk of being on the newer version this is fine

Reply:
It is a good point, `22.x` LTS starts on 2024-10-29 until 2027-04-30.
https://nodejs.org/en/about/previous-releases

Reply:
close enough in that case :D